app/assets/stylesheets/govuk_publishing_components/components/_cookie-banner.scss

```diff
@@ -0,0 +1,19 @@
+$govuk-cookie-banner-background: #d5e8f3;
```

Comment:
Should we use a variable for this from GOVUK Frontend? Or is it a unique colour?

Reply:
Unique colour unfortunately..